src/node/build/buildPluginHtml.ts

```diff
@@ -67,11 +69,35 @@ export const createBuildHtmlPlugin = async (
     }
   }
 
+  const injectSystemJs = (html: string) => {
+    let tag: string
+    if (useSystemJs === false) {
+      return html
+    } else if (useSystemJs === true) {
+      const systemJsRuntime = fs.readFileSync(
+        require.resolve('systemjs/dist/s.min.js'),
+        'utf8'
+      )
+      tag = `<script>${systemJsRuntime}</script>`
+    } else if (typeof useSystemJs === 'string') {
+      tag = `<script src="${useSystemJs}"></script>`
+    } else {
+      return html
+    }
+    if (/<\/body>/.test(html)) {
```

Comment:
This should be targeting `head` instead of `body`. When you rebase, you'll see that's how Vite works now.